lasagne/tests/test_objectives.py

```diff
@@ -153,17 +153,17 @@ def test_binary_hinge_loss(colvect):
 
 
 @pytest.mark.parametrize('colvect', (False, True))
-def test_huber_loss(colvect):
+@pytest.mark.parametrize('delta', (0.5, 1.0))
+def test_huber_loss(colvect, delta):
     from lasagne.objectives import huber_loss
-    delta = [0.5, 1.0]
     if not colvect:
         a, b = theano.tensor.matrices('ab')
-        l1 = huber_loss(a, b, delta[0])
-        l2 = huber_loss(a, b, delta[1])
+        l1 = huber_loss(a, b, delta)
+        l2 = huber_loss(a, b, delta)
```

Comment:
Ah! Yeah, sorry. I rushed through that PR. I will check every comment again before the next commit.